docs/components/RichTextView.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+
+import Example from "./Example";
+import PropDocumentation from "./PropDocumentation";
+import View from "./View";
+import {RichText} from "src";
+
+export default function DateInputView() {
+  return (
+    <View title="RichText" sourcePath="src/RichText/RichText.jsx">
+      <p>
+        RichText is a view that will render user entered text as intended, preserving newlines and linkifying urls/emails.
+      </p>
+      <h2>Without using RichText</h2>
+      <Example>
+        <p>{`
+          Do not go gentle into that good night,
+          Old age should burn and rave at close of day;
+          Rage, rage against the dying of the light.
+          
+          - https://www.poets.org/poetsorg/poem/do-not-go-gentle-good-night
+          - dylan.thomas@poems.com
+        `}</p>
+      </Example>
+      <h2>Using RichText</h2>
+      <Example>
+        <RichText
+          text={`
+            Do not go gentle into that good night,
+            Old age should burn and rave at close of day;
+            Rage, rage against the dying of the light.
+            
+            - https://www.poets.org/poetsorg/poem/do-not-go-gentle-good-night
+            - dylan.thomas@poems.com
+          `}
+        />
+      </Example>
+
+      <PropDocumentation
+        availableProps={[
+          {
+            name: "text",
+            type: "String",
+            description: "The text to render",
+            optional: false,
+          },
+          {
+            name: "newTab",
+            type: "Boolean",
+            description: "Should links open in a new tab?",
+            optional: true,
+            defaultValue: "false",
```

Comment:
what do you think about defaulting to true instead of false for this one? I feel like links opening in a new tab is the more expected / user-friendly behaviort